style.css

```diff
@@ -956,6 +956,50 @@ ul {
   padding: 10px 0 0;
 }
 
+.hero .search {
+  overflow: hidden;
+}
+
+.hero .search:before {
```

Comment:
Could move this into the block above with `&:before {`.

Reply:
@ahmed-arbisoft it is compiled form of `scss`, what do want to do with `style.css` ?